src/service-status/translations/Messages_fr_FR.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<translations>
+   <translation id="status_action_renew_date_label">Renouvellement automatique prévu en </translation>
+   <translation id="status_action_resiliation_cancel_label">Annuler la résiliation</translation>
+   <translation id="status_action_wizard_renew">Renouveler</translation>
+   <translation id="status_action_resiliation_date_label">Résiliation prévue le</translation>
+   <translation id="status_action_expiration_date_label">Expire le</translation>
```

Comment:
```suggestion
   <translation id="status_action_expiration_date_label">Expire le  {{expirationDate}}</translation>
```

Reply:
The meaning of sentences make it look like its a continuous sentence.
But this goes as heading and the date comes as description of the tile element as you can see in the snapshot of the ticket MBE-290

Reply:
Thanks for the input. As I said yesterday, this may not work with all languages. We should see with the Content team if this is an issue

Reply:
This has been through CX team in MBE-240 and is cleared. Thanks